driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -126,8 +126,8 @@ object MongoCredential {
    * @param password the user's password
    * @return the credential
    *
-   * @see [[http://docs.mongodb.org/manual/core/authentication/#mongodb-cr-authentication MONGODB-CR]]
-   * @see [[http://docs.mongodb.org/manual/core/authentication/#authentication-scram-sha-1 SCRAM-SHA-1]]
+   * @see [[https://www.mongodb.com/docs/manual/core/authentication/#mongodb-cr-authentication MONGODB-CR]]
+   * @see [[https://www.mongodb.com/docs/manual/core/authentication/#authentication-scram-sha-1 SCRAM-SHA-1]]
```

Comment:
Looks like this links to an old target.
New link suggestion: https://www.mongodb.com/docs/manual/core/security-scram/#std-label-authentication-scram-sha-1